File: app/src/main/java/com/zivapp/notes/util/UtilDate.java
package com.zivapp.notes.util;

import java.text.SimpleDateFormat;
import java.util.Date;

public class UtilDate {

    public static String getCurrentDate() {
        return new SimpleDateFormat("dd MMM yyyy | HH:mm").format(new Date().getTime());
    }

    public static String getGroupDate() {
        return new SimpleDateFormat("HH:mm | dd MMM yyyy").format(new Date().getTime());
    }
}

File: Kotlin/app/src/main/java/com/zivapp/countandnote/util/UtilDate.kt
package com.zivapp.countandnote.util

import java.text.SimpleDateFormat
import java.util.*

class UtilDate {
    companion object {
        fun getCurrentDate(): String {
            return SimpleDateFormat("dd MMM yyyy | HH:mm").format(Date().time)
        }

        fun getGroupDate(): String {
            return SimpleDateFormat("HH:mm | dd MMM").format(Date().time)
        }
    }
}